Startup.cs

```diff
@@ -0,0 +1,29 @@
+public class Startup {
```

Comment:
I don't believe this class will be called. You have a new style Program.cs with top level statements, it doesn't directly have the concept of Startup anymore unless you call it yourself. Unless I'm missing something

I would just avoid the name Startup so as not to confuse with old style.